src/sentry/integrations/slack/actions/notification.py

```diff
@@ -49,6 +49,7 @@ def after(self, event: GroupEvent, state: EventState) -> Generator[CallbackFutur
 
         def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
             rules = [f.rule for f in futures]
+            event = event.for_group(event.group or event.groups[0])
```

Comment:
You should already have a `GroupEvent` here, not sure why this fixes the bug?

I'd prefer not to have this logic at this level of the codebase

Reply:
Hmmmm let me take another look at this and see if I can find an explanation.

Reply:
My guess here is that the test is maybe passing an `Event` rather than a `GroupEvent`

Reply:
Hmm so `event` can be `None` [here](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/issues.py#L261) and it seems like in prod for whatever reason `self.group.get_latest_event()` is returning `None` so `event_for_tags` is also `None`. I updated the test to not pass the event to try to replicate the behavior, but it finds a latest event. I'm confused why that doesn't seem to be happening in prod.

I could refactor `get_color` to take in the `group` instead and do a check for a performance issue type and make the color blue?